### PR TITLE
Add Ghent local site to the site

### PR DESCRIPTION
**Pull request by nvnieuwk**

### CODE REVIEW

**Comment by maxulysse** [comment]
where is the `belgium-ghent-university.md` file?

**Comment by nvnieuwk** [comment]
I haven't made it yet since we don't have any more information right now. I'll add it once we know more (If it's necessary we can leave this PR open for a bit)

**Comment by maxulysse** [comment]
you don't need much information, look at the CRG or italy page for now

**Comment by nvnieuwk** [comment]
Done!

File: src/content/events/2024/hackathon-march-2024/index.md
Comment (mashehu) [inline src/content/events/2024/hackathon-march-2024/index.md]: please sort them alphabetically

**Comment by mashehu** [comment]
@nf-core-bot fix linting

File: src/content/events/2024/hackathon-march-2024/index.md
Comment (nvnieuwk) [reply src/content/events/2024/hackathon-march-2024/index.md]: Done, thanks!

**Comment by nvnieuwk** [comment]
@nf-core-bot fix linting pretty please

**Review by mashehu — APPROVED** [review]

**Comment by nvnieuwk** [comment]
Thanks @mashehu I'll wait for the preview and then I'll merge if it's good :)

**Pull request merged**
